src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -719,6 +719,82 @@ Array [
     },
   },
 ]
+`);
+      });
+
+      it('works if component has no $vnode', async () => {
```

Comment:
It's currently hard to know why there's no `$vnode` in this use case. Can we either comment, or add an assertion that makes it obvious?

Reply:
```suggestion
      it('works if component is at root, and has no $vnode', async () => {
```

is this clearer @sarahdayan ?

Reply:
```suggestion
      it('works when component is at root (and therefore has no $vnode)', async () => {
```